package.json

```diff
@@ -46,14 +46,14 @@
     "govuk-frontend": "^5.7.1"
   },
   "devDependencies": {
-    "sass": "1.77.6",
     "@babel/preset-env": "^7.21.5",
+    "@playwright/browser-chromium": "^1.49.1",
     "adm-zip": "^0.5.5",
     "adr-log": "^2.2.0",
     "ajv": "^8.12.0",
     "autoprefixer": "^9.6.1",
     "babel-plugin-transform-es2015-modules-commonjs": "^6.26.2",
-    "backstopjs": "^6.1.4",
+    "backstopjs": "6.3.25",
```

Comment:
updating to this without switching to playwright and now being on an arm mac was causing test instability